sky/benchmark/benchmark_utils.py

```diff
@@ -168,7 +168,9 @@ def _create_benchmark_bucket() -> Tuple[str, str]:
     bucket_name = f'sky-bench-{uuid.uuid4().hex[:4]}-{getpass.getuser()}'
 
     # Select the bucket type.
-    enabled_clouds = sky_check.get_enabled_clouds(raise_if_no_cloud_access=True)
+    enabled_clouds = storage_lib.get_enabled_storage_clouds(
```

Comment:
Should we rename this function as well: `get_cached_enabled_storage_clouds_or_refresh`?

Reply:
Good point! Renamed.